Source/Particles/WarpXParticleContainer.cpp

```diff
@@ -798,7 +798,7 @@ Real WarpXParticleContainer::sumParticleCharge(bool local) {
     amrex::Real total_charge = 0.0;
 
     const int nLevels = finestLevel();
-    for (int lev = 0; lev < nLevels; ++lev)
+    for (int lev = 0; lev <= nLevels; ++lev)
```

Comment:
Was the code always just skipping the finest level here?

Reply:
Yeap, that was a bug.